docs/en/quick-answers/troubleshooting/browser-integration-fails-linux.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+title: SG Desktop / browser integration fails to launch on Linux
+pagename: browser-integration-fails-linux
+lang: en
+---
+
+# SG Desktop / browser integration fails to launch on Linux
+
+When running the SG Desktop on Linux for the first time, you may experience one of these error messages. If so, please follow the steps below for your specific error to see if that resolves things. If you're still stuck, please don't hesitate to  contact support@shotgunsoftware.com.
+
+###Contents
+- [OPENSSL_1.0.1_EC or HTTPSConnection related issues](#openssl_101_ec-or-httpsconnection-related-issues)
+- [libffi.so.5 related issues](#libffiso5-related-issues)
+- [Certificate validation failed related issues](#certificate-validation-failed-related-issues)
+- [Incompatible Qt versions](#incompatible-qt-versions)
+
+## OPENSSL_1.0.1_EC or HTTPSConnection related issues
+
+**Errors**
+
+```
+importing '/opt/Shotgun/Resources/Python/tk-framework-desktopstartup/python/server/resources/python/dist/linux/cryptography/_Cryptography_cffi_36a40ff0x2bad1bae.so':
+ /opt/Shotgun/Resources/Python/tk-framework-desktopstartup/python/server/resources/python/dist/linux/cryptography/_Cryptography_cffi_36a40ff0x2bad1bae.so: symbol ECDSA_OpenSSL, version OPENSSL_1.0.1_EC not defined in file libcrypto.so.10 with link time reference
+AttributeError: 'module' object has no attribute 'HTTPSConnection'
+```
+
+**Solution**
+
+You need to install OpenSSL. To do so, run the following command as an administrator:
+
+```
+$ yum install openssl
+```
+
+## libffi.so.5 related issues
+
+**Error**
+
+```
+Browser Integration failed to start. It will not be available if you continue.
+libffi.so.5: cannot open shared object file: No such file or directory
+```
+
+**Solution**
+
+You need to install libffi. To do so, run the following command as an administrator:
+
+```
+yum install libffi
+```
+
+If you've installed libffi and it still doesn't work, try linking the following and then re-launching the SG Desktop:
+
+```
+sudo ln -s /usr/lib64/libffi.so.6.0.1 /usr/lib64/libffi.so.5
+```
+
+Some users have reported success with the above. Others still have issues. The latest version of SG Desktop added some additional dependencies with the web socket server which we're currently looking into. 
+
+## Certificate validation failed related issues
+
+**Possible errors**
+
+```
+Browser Integration failed to start. It will not be available if you continue.
+Error: There was a problem validating if the certificate was installed. 
+certutil: function failed: SEC_ERROR_BAD_DATABASE: security library: bad database.
+```
+
+**Solution** 
+
+If you have Google Chrome installed on your computer, launch it and then relaunch the SG Desktop. If you still have the issue, please contact support@shotgunsoftware.com.
+
+If you don't have Chrome, open a terminal and look for a folder named .pki in your home folder.
+
+```
+ls -al $HOME/.pki/nssdb
```

Comment:
I feel like there's a sentence missing here that would bridge a leap in logic. We're asking them to find a `.pki` folder, but then we're doing an `ls` on `.pki/nssdb`—which is the one we're actually interested in? How about just, "If you don't have Chrome, open a terminal and run the following command:" ?

Reply:
I don't really understand what this is doing so I'll follow your lead.